Day-11-Quiz-App/index.html

```diff
@@ -113,13 +119,93 @@ <h2>Custom Quizzes (Work in progress)</h2>
             customQuizContainer.style.display = "none";
 
         });
+
         quizElement.addEventListener("QuizBuilder:quit", () => {
+            // TODO: pass data of quiz that was just saved along in event, so that we only need to add one more button for the new quiz instead of rebuilding all buttons.
             quizChooser.style.display = "block";
             customQuizContainer.style.display = "block";
+            // TODO: implement dedicated QuizBuilder:save event, as we do not need to rebuild the buttons if no quiz was created.
+            displayCustomQuizElements();

```

Comment:
Here you are calling `displayCustomQuizElements` in line 128, while that variable is only defined in line 131 (afterwards.

If you try to quit your `QuizBuilder` right now and this EventListener is executed, JavaScript will throw an error that `displayCustomQuizElements` is not defined.

If you want a function to be available regardless of the place where it is defined, then you shouldn't define it as a `const´, but using the `function` keyword, like so:
```javascript
function displayCustomQuizElements() {
    displayCustomQuizButtons(getCustomQuizzesFromLocalStorage());
    displaySelectForEditingAndDeletingCustomQuizzes(getCustomQuizzesFromLocalStorage());
}
```

Functions that are created with `function` can be at the very end of a file but still be called above its definition. This can be very handy and should be used in this case.